examples/util.py

```diff
@@ -1,5 +1,6 @@
+from __future__ import print_function, division
 import gzip
-import cPickle as pickle
+import pickle
```

Comment:
That's quite a bit slower, though. It probably won't make much of a difference for MNIST, but I'd prefer a `try ... except` clause.